_posts/en/newsletters/2019-06-26-newsletter.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 'Bitcoin Optech Newsletter #52'
+permalink: /en/newsletters/2019/06/26/
+name: 2019-06-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a pending disclosure of minor
+vulnerabilities for older Bitcoin Core releases, suggests continued
+testing of RCs for LN software, and describes a proposed technique to
+make Bitcoin Core nodes a bit more resistant to eclipse attacks.  Also
+included are our regular sections on bech32 sending support, popular StackExchange topics, and notable
+changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Update Bitcoin Core to at least 0.17.1:** two minor vulnerabilities
+  that affect older releases of Bitcoin Core are
+  [scheduled][vulnerabilities announce] to be disclosed around August
+  1st.  Neither vulnerability affects version 0.17.1 (released 25
+  December 2018) or later versions.  We express our gratitude to Bitcoin
+  Core contributor PracticalSwift for reporting the vulnerabilities.
```

Comment:
I think this is styled practicalswift:

- https://github.com/practicalswift
- https://twitter.com/practicalswift

Reply:
Correct! :-)

You can include a link to https://twitter.com/practicalswift if anyone wants to follow up.